Finix/Presentation/Components/WalletCard/LargeWalletCardView.swift

```diff
@@ -0,0 +1,73 @@
+// Created by Valerie N. Prinz on 18/11/2024.
+
+import SwiftUI
+
+extension LargeWalletCardView {
+    struct Model {
+        let walletName: String
+        let walletBalance: String
+        let styleColor: Color
+        let currencyIsoCode: String
+    }
+}
+
+struct LargeWalletCardView: View {
+    let model: Model
+    
+    var body: some View {
+        VStack(alignment: .leading) {
```

Comment:
Again, default spacing in stacks is not 0.

```suggestion
        VStack(alignment: .leading, spacing: 0) {
```

Look at the distances between labels:

* with default spacing:
<img width="256" alt="image" src="https://github.com/user-attachments/assets/e483f4f1-5758-41f3-a9ed-2458854cd068">


* with 0 spacing:
<img width="256" alt="image" src="https://github.com/user-attachments/assets/669d579f-5497-4a29-982d-cc7ec1eb24be">


* in Figma:
<img width="256" alt="image" src="https://github.com/user-attachments/assets/df0bd1ef-53b3-4ce9-ba23-0af9e372e196">